frontend/src/app/patients/PatientForm.jsx

```diff
@@ -182,6 +183,14 @@ const PatientForm = (props) => {
         (error) => console.error(error)
       );
     }
+    showNotification(
+      toast,
+      <Alert
+        type="success"
+        title="Patient Saved"
+        body="Patient information has been saved."
+      />
+    );
```

Comment:
I think this will show even when there is an error. `showNotification` should be called on the success promise handlers for `updatePatient` and `addPatient` line 177 an 182 respectively

Reply:
Good catch. I'll update the page to use the user-specific term for "patient" and also spew a generic error on failure since right now nothing happens.